gamestonk_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -748,12 +748,39 @@ def call_dcf(self, other_args: List[str]):
             default=False,
             help="Confirms that the numbers provided are accurate.",
         )
+        parser.add_argument(
+            "--no-ratios",
+            action="store_false",
+            dest="ratios",
+            default=True,
+            help="Removes ratios from DCF.",
+        )
+        parser.add_argument(
+            "-p" "--prediction",
+            type=int,
+            dest="prediction",
+            default=10,
+            help="Number of years to predict before using terminal value.",
+        )
+        parser.add_argument(
+            "-s" "--sisters",
+            type=int,
+            dest="sisters",
+            default=3,
```

Comment:
I would increase this or remove the limit. Especially if you are looking for companies in a specific industry with a specific market cap. It would be a shame that one of the big players is excluded because of this limitation.